frontend/pages/index.vue

```diff
@@ -59,132 +59,122 @@ export default Vue.extend({
   },
   data() {
     return {
-      homeSections: [] as HomeSection[],
       carouselItems: [] as BaseItemDto[]
     };
   },
-  async fetch() {
-    const validSections = ['resume', 'resumeaudio', 'upnext', 'latestmedia'];
-
-    // Filter for valid sections in Jellyfin Vue
-    // TODO: Implement custom section order
-    let homeSectionsArray = pickBy(
-      // @ts-expect-error - No typings for this
-      this.clientSettings.CustomPrefs,
-      (value: string, key: string) => {
-        return (
-          value &&
-          validSections.includes(value) &&
-          key.startsWith('homesection')
-        );
-      }
-    );
-
-    if (!Object.keys(homeSectionsArray).length) {
-      homeSectionsArray = {
-        homeSection0: 'librarytiles',
-        homeSection1: 'resume',
-        homeSection2: 'resumeaudio',
-        homeSection3: 'upnext',
-        homeSection4: 'latestmedia'
-      };
-    }
-
-    // Convert to an array
-    homeSectionsArray = Object.values(homeSectionsArray);
-
-    const homeSections: HomeSection[] = [];
-
-    for (const homeSection of homeSectionsArray as Array<string>) {
-      switch (homeSection) {
-        case 'librarytiles': {
-          homeSections.push({
-            name: 'libraries',
-            libraryId: '',
-            shape: CardShapes.Thumb,
-            type: 'libraries'
-          });
-          break;
+  head() {
+    return {
+      title: this.page.title
+    };
+  },
+  computed: {
+    ...mapStores(clientSettingsStore, pageStore, userViewsStore),
+    homeSections(): HomeSection[] {
+      const validSections = ['resume', 'resumeaudio', 'upnext', 'latestmedia'];
```

Comment:
I think this kind of "hardcoded" value lives better outside the Vue component, so it's easier to locate later one (no need to revisit the whole component).

What do you think about adding it at line 37?

Reply:
No problem, done :)